src/app/sport/[sport]/page.tsx

```diff
@@ -0,0 +1,60 @@
+import prisma from '@db/prismaSingleton'
+
+import SelectGrid from '@components/SelectGrid'
+import TopBar from '@components/TopBar'
+
+type PageProps = {
+  params: { sport: string },
+}
+
+const Page = async (props: PageProps) => {
+  const {
+    params: { sport: sportKey },
+  } = props
+
+  const sport = await prisma.team.findUnique({
+    where: {
+      key: sportKey,
+    },
+  })
+  const { id: sportId, title: sportTitle } = sport || {}
+
+  const competitions = await prisma.competition.findMany({
+    where: {
+      sportId,
+    },
+  })
+
+  const hasCompetitions = !!competitions?.length
+
+  return (
+    <div className="max-w-column">
+      <TopBar />
+
+      <div className="p-2 w-full">
+        {!hasCompetitions && (
+          <div>
+            {sportTitle?.toUpperCase()} Coming Soon
```

Comment:
if we run the query above we wouldn't run the query for sport, so i would replace all references to `sportTitle` with generic titles like "Coming Soon", "Select a Competition"

Reply:
have replied about this above